arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -957,6 +1064,21 @@ IResearchAnalyzerFeature::AnalyzerPool::ptr IResearchAnalyzerFeature::get( // fi
     irs::string_ref const& name // analyzer name
 ) const noexcept {
   try {
+    auto split = splitAnalyzerName(name);
+
+    if (!split.first.null()) { // do not trigger load for static-analyzer requests
+      auto res = // load analyzers for database
+        const_cast<IResearchAnalyzerFeature*>(this)->loadAnalyzers(split.first);
+
+      if (!res.ok()) {
+        LOG_TOPIC(,WARN, arangodb::iresearch::TOPIC)
```

Comment:
this looks like as if it will not compile. Log ID is missing here.

Reply:
compiles and runs perfectly

Reply:
However, this is not how the LOG_TOPIC macro is supposed to be used.